insights/parsers/hammer_compute_resource_list.py

```diff
@@ -0,0 +1,56 @@
+"""
+HammerComputeResourceList - command ``hammer --interactive 0 --output json compute-resource list``
+===================================================
+
+This parser reads the compute resource list of a Satellite server using hammer in JSON format.
+It relies on the root user running the command and hammer to authenticate based on the hammer configuration file.
+If the command is unable to authenticate then no compute resources will be shown and an erro flag will be recorded in the parser
+
+Sample output from the ``hammer --interactive 0 --output json compute-resource list`` command::
+
+    [
+  {
```

Comment:
One more block (4 blank spaces) should be added to the following lines  (to L27) to keep the alignment for the `::` section.